<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QDialog" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>667</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Insight Qt Slicer</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
     <item>
      <widget class="QtGlSliceView" name="OpenGlWindow" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="3">
        <widget class="QSpinBox" name="SliceNum">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="Slice">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="PositionX">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PositionY">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PositionZ">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TextLabel1">
       <property name="text">
        <string>=</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PixelValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="ZoomIn">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Zoom In</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ZoomOut">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Zoom Out</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="IntensityMinLabel">
       <property name="text">
        <string>Int. Min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="IntensityMinDisplay"/>
     </item>
     <item>
      <widget class="QSlider" name="IntensityMin">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="IntensityMaxLabel">
       <property name="text">
        <string>Int. Max</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="IntensityMaxDisplay"/>
     </item>
     <item>
      <widget class="QSlider" name="IntensityMax">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonHelp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="windowTitle">
        <string/>
       </property>
       <property name="text">
        <string>&amp;Quit</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtGlSliceView</class>
   <extends>QWidget</extends>
   <header>QtGlSliceView.h</header>
   <slots>
    <signal>XValueChanged(double)</signal>
    <signal>YValueChanged(double)</signal>
    <signal>ZValueChanged(double)</signal>
    <signal>PixelValueChanged(double)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonHelp</tabstop>
  <tabstop>buttonOk</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Slice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenGlWindow</receiver>
   <slot>ChangeSlice(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1018</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntensityMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenGlWindow</receiver>
   <slot>IntensityMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>671</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntensityMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenGlWindow</receiver>
   <slot>IntensityMin(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>643</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ZoomIn</sender>
   <signal>clicked()</signal>
   <receiver>OpenGlWindow</receiver>
   <slot>ZoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>903</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ZoomOut</sender>
   <signal>clicked()</signal>
   <receiver>OpenGlWindow</receiver>
   <slot>ZoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>967</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SliceNum</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1018</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>993</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliceNum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slice</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>993</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>1018</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntensityMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IntensityMaxDisplay</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>671</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>671</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntensityMaxDisplay</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IntensityMax</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>671</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>671</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntensityMinDisplay</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IntensityMin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>643</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntensityMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IntensityMinDisplay</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>643</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>Gui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>960</x>
     <y>701</y>
    </hint>
    <hint type="destinationlabel">
     <x>734</x>
     <y>667</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenGlWindow</sender>
   <signal>XValueChanged(double)</signal>
   <receiver>PositionX</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>622</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenGlWindow</sender>
   <signal>YValueChanged(double)</signal>
   <receiver>PositionY</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenGlWindow</sender>
   <signal>ZValueChanged(double)</signal>
   <receiver>PositionZ</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>623</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenGlWindow</sender>
   <signal>PixelValueChanged(double)</signal>
   <receiver>PixelValue</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>625</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
